Sources/Hummingbird/Files/FileIO.swift

```diff
@@ -123,6 +123,7 @@ public struct FileIO: Sendable {
                     fileOffset = range.index(fileOffset, offsetBy: bytesToRead)
                     try await writer.write(buffer)
                 }
+                try await writer.finish(nil)
```

Comment:
Is this something we can test in a unit test?

Reply:
I thought we did, obviously not. I'll add a test in a separate PR.